### ISSUE TITLE
Execution Error

### ISSUE THREAD

**Issue by shahrukhkhan489** [timegalore/MotionDetectionAR#1]
Hey i tried to execute the code but found this problem.
"block10_obj is missing"
![ScreenShot](https://cloud.githubusercontent.com/assets/12380350/7553987/20f983ee-f731-11e4-9d2a-728a95325277.png)

**Comment by Kokika** [comment]
Hali,

There are any object in res/raw? It would be need one "block10.obj" .
But for me this application can't work the latest Rajawali. So i wrote a
another. My application has faster detection.

Üdv:
Kornél

2015-05-10 12:57 GMT+02:00 shahrukhkhan489 notifications@github.com:

**Comment by shahrukhkhan489** [comment]
yes "obj10_obj.obj" file is not there in raw folder. There is a "obj_mtl.mtl" file in that folaseder. Could you upload a complete working project please. I need it a lot.

**Comment by Kokika** [comment]
Rewrite for this: R.raw.block10_mtl
So the part:
LoaderOBJ objParser = new LoaderOBJ(mContext.getResources(),
mTextureManager, R.raw.block10_mtl);
Than you can make apk but my project did't work. Something wrong with the
object rendering.

Good luck.
(And also you need red.jpg in your phone because the app read it and if
did't find it will make error. red.jpg is one picture of your skin or else
thing...)

BB.

2015-05-10 17:09 GMT+02:00 shahrukhkhan489 notifications@github.com:

**Comment by shahrukhkhan489** [comment]
Yes I understand.... but where am i supposed to put red.jpg.
Is it directly in my sdcard or phone memory
Or is it in res->drawable

**Comment by Kokika** [comment]
somewhere sdcard, one of the top mappa or where programs resource place
could find

2015-05-11 13:33 GMT+02:00 shahrukhkhan489 notifications@github.com:
